content/tutorials/r_citations_markdown/index.Rmd

```diff
@@ -152,7 +153,7 @@ The [ymlthis field guide](https://ymlthis.r-lib.org/articles/yaml-fieldguide.htm
 :   Specify one or more Zotero libraries.
     Or `false` to turn of Zotero library entirely.
 
-Here is an example of a valid YAML with all citation arguments:
+Here is an example of a valid YAML with all citation arguments ("yes" and "no" serve as alternatives to `true` and `false`):
```

Comment:
Does not work for `zotero` key at least - see https://github.com/inbo/tutorials/pull/269#discussion_r666181955. So best stay with `true` or `false`, unquoted, for the VME specific ones.

Reply:
The problem is that I had to add zotero key via `yml_toplevel()` which does not validate values and `yml_citation()` does not include a zotero argument.

Reply:
An alternative would be to just hardcode the YAML header (copy from `index.md` and update where needed, since the tutorial does not demonstrate `ymlthis`.

Reply:
Solved it 5b46113 😉